storybook/src/components/form/form.css-template.ts

```diff
@@ -20,25 +20,21 @@ export const cssForm: ComponentImplementation<Form<TemplateResult>> = {
       return html`
         <form class=${ifDefined(mode === "horizontal" ? "form-horizontal" : undefined)}>
           ${asteriskExplanation === "top" || asteriskExplanation === "both" ? asteriskExplanationTemplate() : nothing}
-          <fieldset>
-            <legend>
-              ${legendHeading === "h1" // if
-                ? html`<h1>${legend}</h1>`
-                : legendHeading === "h2" // else if
-                  ? html`<h2>${legend}</h2>`
-                  : legendHeading === "h3" // else if
-                    ? html`<h3>${legend}</h3>`
-                    : legendHeading === "h4" // else if
-                      ? html`<h4>${legend}</h4>`
-                      : legendHeading === "h5" // else if
-                        ? html`<h5>${legend}</h5>`
-                        : legendHeading === "h6" // else if
-                          ? html`<h6>${legend}</h6>`
-                          : html`${legend}`}
-            </legend>
-            ${"_$litType$" in formGroups ? formGroups : formGroups.map((formGroup) => formGroupTemplate(formGroup))}
-            ${formButtons ? formButtonsTemplate(formButtons) : nothing}
-          </fieldset>
+          ${formHeading === "h1" // if
+            ? html`<h1>${formTitle}</h1>`
+            : formHeading === "h2" // else if
+              ? html`<h2>${formTitle}</h2>`
+              : formHeading === "h3" // else if
+                ? html`<h3>${formTitle}</h3>`
+                : formHeading === "h4" // else if
+                  ? html`<h4>${formTitle}</h4>`
+                  : formHeading === "h5" // else if
+                    ? html`<h5>${formTitle}</h5>`
+                    : formHeading === "h6" // else if
+                      ? html`<h6>${formTitle}</h6>`
+                      : html`${formTitle}`}
```

Comment:
Dit "oogt" soms wat kaal. En als ik naar de source kijk van de form dan zit de "losse" in de form.  Is dat wenselijk.

<img width="348" alt="Screenshot 2024-08-13 at 14 05 53" src="https://github.com/user-attachments/assets/d7abfd8d-58ff-46a1-a355-bcf9e896601f">

Reply:
Kan wel op zich, maar kan er ook buiten